dist/Pagination/Pagination.js

```diff
@@ -0,0 +1,204 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+/**
+ * Primary UI component for user interaction
+ */
+
+const Pagination = ({
+  variant = 'primary',
+  borderVariant = 'black',
+  size = 'md',
+  rounded = 'md',
+  additionalClasses = [],
+  activeAdditionalClasses = [],
+  prevNextAdditionalClasses = [],
+  containerAdditionalClasses = [],
+  prev = 'Previous',
+  next = 'Next',
+  baseUrl = '/',
+  page = 1,
+  numOfPages = 1
+}) => {
+  const sizeClass = {
+    sm: 'w-6 h-6 md:w-8 md:h-8 text-sm',
+    md: 'w-8 h-8 md:w-10 md:h-10 text-sm md:text-base',
```

Comment:
to be consistent with button, it should be 'text-md' by default, not `base`
```suggestion
    md: 'w-8 h-8 md:w-10 md:h-10 text-sm md:text-md',
```